lizard_auth_server/templates/lizard_auth_server/login.html

```diff
@@ -4,25 +4,27 @@
 {% block subtitle %}{% trans 'Log in' %}{% endblock %}
 
 {% block container %}
-    <div class="content">
-        {% if form.errors %}
+<div class="content">
```

Comment:
Ze springen in die docs de for loop ook niet in.

Zonder de inspring-hulp zie ik vaak niet wat er echt aan de hand is in een template. Als ik *iets* aan een template in iemand anders product moet fixen zet ik eerst al de indentatie "goed" voor de duidelijkheid.